core/src/worker/workflow/mod.rs

```diff
@@ -1441,14 +1445,18 @@ fn prepare_to_ship_activation(wfa: &mut WorkflowActivation) {
 mod tests {
     use super::*;
     use itertools::Itertools;
+    use temporal_sdk_core_protos::coresdk::workflow_activation::SignalWorkflow;
 
     #[test]
     fn jobs_sort() {
         let mut act = WorkflowActivation {
             jobs: vec![
                 WorkflowActivationJob {
                     variant: Some(workflow_activation_job::Variant::SignalWorkflow(
-                        Default::default(),
+                        SignalWorkflow {
+                            signal_name: "1".to_string(),
+                            ..Default::default()
```

Comment:
What happens if the application code also uses the signal_name "1", do we need to reserve some signal names?

Reply:
Hmm? This is just test code.

Reply:
Sorry...Duh...